### ISSUE TITLE
Train result is bad with custom data.

### ISSUE THREAD

**Issue by xuduo18311199384** [wutong16/Voxurf#29]
I use 70 images from diff angles to train. But train result is bad. 
![2023-09-07 10-18-20 的屏幕截图](https://github.com/wutong16/Voxurf/assets/30895764/44d74bf9-08de-4524-ab28-2130edb3ba72)
![2023-09-07 10-18-28 的屏幕截图](https://github.com/wutong16/Voxurf/assets/30895764/0394d08a-66b7-48a9-93f5-4a458b723665)
![2023-09-07 10-20-54 的屏幕截图](https://github.com/wutong16/Voxurf/assets/30895764/1afa1512-56a8-4797-847c-9106a458c3e5)
![2023-09-07 10-21-11 的屏幕截图](https://github.com/wutong16/Voxurf/assets/30895764/c8350668-c854-4571-bcd3-6b0bd7abc114)

**Comment by wutong16** [comment]
Hi. First, please try to extract the mesh with a higher resolution like 512. The current resolution appears to be rather low, and this adjustment may yield improved results. 
If the outcome remains unsatisfactory, please investigate whether there is any slight movement in the foreground object during the video capture process, which may lead to un-consistency under the assumption of a static scene.
Further more, it seems that the case is run under the without-mask setting, please also try the with-mask setting and see if the result gets better.

**Comment by xuduo18311199384** [comment]
Thanks for your reply.
First. My resolution config is 128. I will change it.
Next. I have got /image_rmbg and /mask. I train with "bash single_runner.sh ./configs/custom_e2e data /voxurf", but it seems not without mask. could you tell me how can i use mask?

**Comment by wutong16** [comment]
Yes, this should have already been executed with the 'wmask' setting. The current PSNR, even for the background, appears to be quite low. I'm uncertain whether this is attributed to noise in the SFM camera pose estimation. BTW, have you observed any improvement in the mesh quality when extracting it at a resolution of 512?
